src/native/Svg.tsx

```diff
@@ -99,18 +93,17 @@ class NativeSvg extends Component<RequiredIContentLoaderProps, State> {
 
     return (
       <Svg
-        viewBox={`0 0 ${width} ${height}`}
+        style={composedStyle}
         width={width}
         height={height}
-        preserveAspectRatio="none"
-        style={composedStyle}
+        viewBox={viewBox}
         {...props}
       >
         <Rect
           x="0"
           y="0"
-          width={width}
-          height={height}
+          width={Number(vbWidth)}
+          height={Number(vbHeight)}
```

Comment:
Hey, it's me again :)
This implementation based on my previous Pull request is actually not a great idea.
When Using `Number`or `parseInt`it become impossible to use `width='100%'` or `height='100%'`

I really think that react-content-loader should be *transparent* and we should be able to switch from `Svg` to `ContentLoader` without seing any difference in terms of size/ratio etc... 

Is there any limitation with forwarding `width` `height` and `viewBox` ?

I did it in this other commit if you want to try : https://github.com/ACHP/react-content-loader/commit/0eeff41bbfa6b05604685eae7f76001fe202507c
( the commit contains build file as well for test purpose sry :/ )

Reply:
Hey, I'm glad you're reviewing this PR, let me know if you have any else thoughts about these change.

---

So I just got this bug on my tests, I'm going to update it.

Reply:
I see you use `parseInt`now, you'll have the exact same problem ...
```javascript
Number('100%') // NaN
parseInt('100%') // 100
```

Sorry to ask you **again** but : **is there any logical reason that force you to mess with size and viewbox, instead of just forwarding them ??**

Reply:
tl;dr
After some tests and researches, I could see that React Native has a **different behavior** than on the web. And, I was not able to find a way to standardize their behavior. 

As my goal always was to have the same API between the platforms, I still don't know which way I should go, but probably it will not have the same API anymore.

---

So, the main reason that I choose to generate a default `viewBox`, because it is the easiest way to deliver a package/SVG ready to use, without too many configurations. By default, it is already responsive and its behavior is more predictable.

Check the followings loadings, both are on a container with width `300px`:

| With `viewBox`: works fine, without any prop:  | Without `viewBox`: crop the loading:  | 
|---|---|
| <img width="223" alt="Screenshot 2020-01-20 at 21 00 43" src="https://user-images.githubusercontent.com/4838076/72757496-0749da80-3bc8-11ea-8558-1e81cceffcae.png">  |  <img width="223" alt="Screenshot 2020-01-20 at 21 00 58" src="https://user-images.githubusercontent.com/4838076/72757472-fb5e1880-3bc7-11ea-8606-53a127e44853.png"> |

Also, IMHO it is the right way to write SVGs, as width and height only defines the size of the document, but doesn't the graphic itself. So it is necessary to say what is coordinates and sizes of graphics into the document/SVG. Probably I would have more issues asking me how to turn it responsible than how to remove the `viewBox` attr.

**But,** on Native everything is different and the style (width 100%) **doesn't work** the same way than on the web:

| Regular way, `viewBox` generated on component  | With inline `viewBox` and attr width 100% |
|---|---|
| <img width="582" alt="Screenshot 2020-01-20 at 21 05 02" src="https://user-images.githubusercontent.com/4838076/72757733-a7076880-3bc8-11ea-9655-64202b2fb11d.png"> | <img width="582" alt="Screenshot 2020-01-20 at 21 09 44" src="https://user-images.githubusercontent.com/4838076/72757933-3745ad80-3bc9-11ea-9e8f-4c26d3e9895e.png"> |

Reply:
Thank you for this long detailed response :) 

I do agree on the fact that `width`and `height` attributes are used to defines the `visible` size of the document and `viewbox` the 'box' in which the svg is contained. And that's exactly why I think that it should be up to the person who use the SVG to set the viewbox and this lib should not mess with it.
To be clear, to me, all your Instagram,Facebook,BulletList svgs... should all have a viewBox `attribute` and the end user could decide to overwrite it or not, and just play width the `width` and `height` to make them responsive.

I think your solution work well on the web because you're not using `width` and `height`attribute to make the responsiveness of your component but css instead and this is impossible on RN.

> it is already responsive and its behavior is more predictable

I disagree with that, it's *maybe easier* for someone who have never played with svgs before, but it's actually quite the opposite for anyone else.

> Probably I would have more issues asking me how to turn it responsible than how to remove the viewBox attr.

If people ask you how to make it responsive, the answer should be "exactly like any other svg". IMHO it should not be the responsability of this library to implement an "easier svg responsive system."

But maybe I'm wrong ... would be nice to have more people participating in this conversation.
Anyway, thank you for your time :)
At the end of the day, it's up to you, it's your library ;)